data-node/src/main/java/org/graylog/datanode/configuration/OpensearchConfigurationProvider.java

```diff
@@ -121,9 +124,11 @@ public OpensearchConfiguration get() {
                     localConfiguration.getOpensearchTransportPort(),
                     localConfiguration.getClustername(),
                     localConfiguration.getDatanodeNodeName(),
-                    List.of(),
+                    List.of("cluster_manager", "data", "ingest", "remote_cluster_client", "search"),
```

Comment:
if we have a list and not the default, we should make it a config setting imho

Reply:
if we ever have the need to change it ourselves (e.g. via support) I guess it should not mean that we have to release a bugfix, right? I don't see the necessity to advertise it (as we do with other config options) but I guess, it should be there?

Reply:
You are right, I have made it configurable.